src/test/java/api/requests/scenarios/RequestPolicyTests.java

```diff
@@ -0,0 +1,378 @@
+    final String recallRequestPolicy = requestPoliciesFixture.recallRequestPolicy().getId().toString();
+
+    //This rule is set up to show that the fallback policy won't be used but the patronGroup policy g is used instead.
+    //The patronGroup policy allows undergraduate students to place a request on any material type, loan or notice type.
+    final String rules = String.join("\n",
+      "priority: t, s, c, b, a, m, g",
+      "fallback-policy : l " + anyLoanPolicy + " r " + anyRequestPolicy + " n " + anyNoticePolicy + "\n",
+      "g " + undergradPatronGroup + ": l " + anyLoanPolicy + " r " + recallRequestPolicy +" n " + anyNoticePolicy
+    );
+
+    setRules(rules);
+
+    //setting up a checked-out library item to perform recall
+    final IndividualResource checkedOutItem = itemsFixture.basedUponSmallAngryPlanet();
+    loansFixture.checkOut(checkedOutItem, usersFixture.jessica());
+
+    //an undergraduate student attempts to place a Recall request
+    final IndividualResource recallRequest = requestsClient.create(new RequestBuilder()
+      .recall()
+      .forItem(checkedOutItem)
+      .withPickupServicePointId(requestPickupServicePoint.getId())
+      .by(usersFixture.undergradHenry()));
+
+    JsonObject requestedItem = recallRequest.getJson().getJsonObject("item");
+    assertThat(recallRequest.getJson().getString("requestType"), is(RequestType.RECALL.getValue()));
+    assertThat(requestedItem.getString("status"), is(ItemStatus.CHECKED_OUT.getValue()));
+    assertThat(recallRequest.getJson().getString("status"), is(RequestStatus.OPEN_NOT_YET_FILLED.getValue()));
+  }
+
+  @Test
+  public void cannotCreateRecallRequestsWithRequestPolicyNotAllowingRecalls()
+    throws InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+
+    final String undergradPatronGroupPolicy = patronGroupsFixture.undergrad().getId().toString();
+    final String anyNoticePolicy = noticePoliciesFixture.activeNotice().getId().toString();
+    final String anyLoanPolicy = loanPoliciesFixture.canCirculateRolling().getId().toString();
+    final String anyRequestPolicy = requestPoliciesFixture.allowAllRequestPolicy().getId().toString();
+
+    ArrayList<RequestType> allowedRequestTypes = new ArrayList<>();
+    allowedRequestTypes.add(RequestType.HOLD);
+    allowedRequestTypes.add(RequestType.PAGE);
+    final String noRecallRequestPolicy = requestPoliciesFixture.customRequestPolicy(allowedRequestTypes,
+                                          "All But Recall", "All but Recall request policy").getId().toString();
+
+    //This rule is set up to show that the fallback policy won't be used but the patronGroup policy g is used instead.
+    //The patronGroup policy allows undergraduate students to place a request on any material type, loan or notice type, on any request type
+    //except for RECALL
+    final String rules = String.join("\n",
```

Comment:
At some point, we really need a builder for circulation rules, however this story is not going to fit that scope.